gen3-integration-tests/services/gen3workflows.py

```diff
@@ -0,0 +1,133 @@
+from typing import Dict, Tuple
+
+import boto3
+import botocore.exceptions
+import pytest
+import requests
+from gen3.auth import Gen3Auth
+from utils import logger
+
+
+class Gen3Workflow:
+    def __init__(self):
+        self.BASE_URL = f"{pytest.root_url}"
+        self.SERVICE_URL = "/workflows"
+        self.S3_ENDPOINT_URL = f"{self.BASE_URL}{self.SERVICE_URL}/s3"
+
+    def _get_access_token(self, user: str = "main_account") -> str:
+        """Helper function to retrieve an access token."""
+        auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
+        return auth.get_access_token()
+
+    def _get_s3_client(self, access_token: str):
+        """Creates and returns an S3 client."""
+        return boto3.client(
+            service_name="s3",
+            aws_access_key_id=access_token,
+            aws_secret_access_key="N/A",  # pragma: allowlist secret
```

Comment:
best practice is to add it to the baseline instead of `# pragma: allowlist secret`